OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -106,7 +106,8 @@ EditorActorPreview Add(string id, ActorReference reference, bool initialSetup =
 
 			var preview = new EditorActorPreview(worldRenderer, id, reference, owner);
 			previews.Add(preview);
-			screenMap.Add(preview, preview.Bounds);
+			if (!preview.Bounds.IsEmpty)
```

Comment:
The exception thrown is already sensible thanks to the first 2 commits. It just shouldn't throw here. We already silently skip the actors from the editor actor selector.